packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_distribution_repository/__brick__/crypto.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
```

Comment:
Please provide help option of the command so that the developers can can familiar with the usage of this command without the need to read the actual script. Also 
- I'm not sure that we do not violate the SRP by having one script for encrypting and decrypting
- Is crypto a common name ?

Reply:
For the first point, didn't we mention in #681 to unify those scripts in the first place?

As per the second point, that would be a common name in cryptography, but for the purposes of our needs for encoding/decoding files I wan't sure how to name it.

Reply:
Apologies for the confusion, by unify I meant to make them consistent.

The developers should be able to

1. Encrypt all files
2. Encrypt a specific file
3. Decrypt all files

If you think that we can use one script for all the the above operations I'm ok as long is properly documented